dev3/2024-07-02/example-metadata/min-specimen-biosample.json

```diff
@@ -0,0 +1,74 @@
+{
+    "@context": [
+        "https://w3id.org/ro/crate/1.1/context",
+        {
+            "organism_classification": "https://schema.org/taxonomicRange",
+            "BioChemEntity": "https://schema.org/BioChemEntity",
+            "obo": "http://purl.obolibrary.org/obo/",
+            "acquisiton_method": {
+                "@reverse": "https://schema.org/result",
+                "@type": "@id"
+            },
+            "biological_entity": "https://schema.org/about",
+            "preparation_method": "https://www.wikidata.org/wiki/Property:P1537"
+        }
+    ],
+    "@graph": [
+        {
+            "@type": "CreativeWork",
+            "@id": "ro-crate-metadata.json",
+            "conformsTo": {
+                "@id": "https://w3id.org/ro/crate/1.1"
+            },
+            "about": {
+                "@id": "./"
+            }
+        },
+        {
+            "@id": "./",
+            "@type": "Dataset",
+            "name": "Light microscopy photo of a fly",
+            "description": "Light microscopy photo of a fruit fly.",
+            "licence": "https://creativecommons.org/licenses/by/4.0/",
+            "hasPart": {
+                "@id": "./dros-mel-image.zarr/"
+            }
+        },
+        {
+            "@id": "./dros-mel-image.zarr/",
+            "@type": "Dataset",
+            "name": "OME-ZARR files",
+            "description": "the ome zarr files of the fly.",
+            "acquisition_method": [
```

Comment:
These could validly be written as:

```
"acquisition_method": "_:b0"
```
(i.e. non-list)?

Reply:
Yes, though i'm always aware that biological images contain a lot of edge cases & I mostly only know that i'm not knowledgeable in this area. E.g. what happens if an image is created via the combination of various imaging techniques?

So in general, i went with values in lists. The RDF users won't see any difference, but I expected JSON users would prefer to always expect a list even if there's occasionally only one element in it.

Reply:
> what happens if an image is created via the combination of various imaging techniques?

Definitely agreed that a multi-answer is possible.

> I expected JSON users would prefer to always expect a list even if there's occasionally only one element in it

This is a good point.

Reply:
> I expected JSON users would prefer to always expect a list even if there's occasionally only one element in it

Noting that this would be a good rule to enshrine in any profile you make if you want to enforce it for your user base, since it's more restrictive than base RO-Crate

Reply:
And on fact this contradicts the "compact" requirement.